subproject-lifecycle.md

```diff
@@ -0,0 +1,82 @@
+# Lifecycle of a subprojects
```

Comment:
Is there anything to do for subproject deletion other then deleting the corresponding yaml entries? If there is, we could include these steps here as well.

Reply:
good idea, I'll add steps for subproject retirement. I think we should archive things instead of deleting.. but this will be probably refined future when we face the need to retire a project. Right now we don't have such need so we don't have a complete picture for what it means for this governance model I think...